src/NuGet.Protocol.Catalog/Models/CatalogPageItem.cs

```diff
@@ -8,8 +8,7 @@ namespace NuGet.Protocol.Catalog.Models
 {
     public class CatalogPageItem
     {
-        [JsonProperty("@id")]
-        public string Url { get; set; }
+        [JsonProperty("@id")] public string Url { get; set; } = "";
```

Comment:
You can hit this button: 
![image](https://user-images.githubusercontent.com/1633368/70357046-e7db3600-1843-11ea-9f0a-e20276c6d119.png)
And offer a "suggestion" like so:

```suggestion
        [JsonProperty("@id")]
        public string Url { get; set; } = "";
```

It copies the line over and you can edit. Then I can just hit "commit"